comp/core/status/component.go

```diff
@@ -21,6 +21,8 @@ const CollectorSection string = "collector"
 type Component interface {
 	// Returns all the agent status information for the format type
 	GetStatus(format string, verbose bool, excludeSection ...string) ([]byte, error)
+	//
+	GetSections() []byte
```

Comment:
I think `GetSections` should return `[]string` and then the api endpoint should convert that to bytes to write it, the function could be used from somewhere else

Reply:
It should probably also return an error while we're at it

Reply:
I had this approach at the begin but move to `[]byte` to match with the other methods.
But I revert my changes in 78a57b45c651eaf0d39f244c85ae3698ae11ab87.

I don't think it require error return because it's just returning a slice.